src/screens/Contact/Contact.js

```diff
@@ -283,6 +285,10 @@ class Contact extends React.Component<Props, State> {
   };
 
   setActiveTab = (activeTab) => {
+    const { logScreenView } = this.props;
+
+    logScreenView(activeTab, 'Contact');
```

Comment:
Just to check: This will be logged only when user presses on a tab. It won't log screen on its first render.